staging/src/k8s.io/client-go/tools/cache/testing/fake_controller_source_test.go

```diff
@@ -93,3 +93,37 @@ func TestRCNumber(t *testing.T) {
 	source.Shutdown()
 	wg.Wait()
 }
+
+func TestRCNumberDisruptive(t *testing.T) {
```

Comment:
comments explaining what this is testing would be helpful to future generations... it's not immediately clear to me how this relates to events from a relist